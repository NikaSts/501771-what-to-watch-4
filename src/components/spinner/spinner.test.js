import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './spinner';


it(`Spinner should render correctly`, () => {
  const tree = renderer
    .create(
        <Spinner />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
