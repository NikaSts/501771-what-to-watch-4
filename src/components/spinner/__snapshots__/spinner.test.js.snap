// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render correctly 1`] = `
<div
  className="loadingio-spinner-spin-ndgslan05n"
>
  <div
    className="ldio-lenc1v6s21e"
  >
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
    <div>
      <div />
    </div>
  </div>
</div>
`;
